### ISSUE TITLE
[confusing] Feedback from `/serverless/observability/apm-transaction-sampling`

### ISSUE THREAD

**Issue by chrisdistasio** [elastic/observability-docs#3279]
## What do you see?

> OpenTelemetry with head-based sampling
Head-based sampling is implemented in the APM agents and SDKs, and requires the sample rate to be propagated between services and the managed intake service. This functionality is not currently supported by OpenTelemetry, which results in inaccurate APM throughput, latency, and error metrics.

## What were you expecting to see?

@AlexanderWert is this correct?  can a user making user of otel not control sampling at all?  thats the way that this reads to me.

**Comment by AlexanderWert** [comment]
@chrisdistasio That is correct. Users can control the (head-based) sampling rate, but the sampling rate is not being propagated to the backend (i.e. MIS), so there's no way for the MIS to extrapolate metrics if head-based sampling rate is < 100%

**Comment by chrisdistasio** [comment]
ok so for otel users on serverless, they cannot use any sampling?

**Comment by AlexanderWert** [comment]
They can use sampling but especially the throughput metric will be wrong. 

Let's say they have 10% with a real throughput of 5000 tpm, what we would show then is ~ 500 tpm.

**Comment by AlexanderWert** [comment]
@chrisdistasio Actually, I just checked that we removed that limitation from this list already:

https://www.elastic.co/guide/en/apm/guide/current/open-telemetry-known-limitations.html

So, maybe that paragraph is outdated. Let me follow-up on this, to clarify the state.
